merlin/schema/io/tensorflow_metadata.py

```diff
@@ -35,69 +35,197 @@
         return TensorflowMetadata(schema)
 
     @classmethod
     def from_json_file(cls, path: str) -> "TensorflowMetadata":
+        """Create a TensorflowMetadata schema object from a JSON file
+
+        Parameters
+        ----------
+        path : str
+            Path to the JSON file to parse
+
+        Returns
+        -------
+        TensorflowMetadata
+            Schema object parsed from JSON file
+
+        """
         return cls.from_json(_read_file(path))
 
     @classmethod
-    def from_proto_text(cls, path_or_proto_text: str) -> "TensorflowMetadata":
+    def from_proto_text(cls, proto_text: str) -> "TensorflowMetadata":
+        """Create a TensorflowMetadata schema object from a Protobuf text string
+
+        Parameters
+        ----------
+        proto_text : str
+            Protobuf text string to parse
+
+        Returns
+        -------
+        TensorflowMetadata
+            Schema object parsed from Protobuf text
+
+        """
         from tensorflow_metadata.proto.v0 import schema_pb2
 
         schema = proto_utils.proto_text_to_better_proto(
-            ProtoSchema(), path_or_proto_text, schema_pb2.Schema()
+            ProtoSchema(), proto_text, schema_pb2.Schema()
         )
 
         return TensorflowMetadata(schema)
 
     @classmethod
     def from_proto_text_file(cls, path: str, file_name="schema.pbtxt") -> "TensorflowMetadata":
+        """Create a TensorflowMetadata schema object from a Protobuf text file
+
+        Parameters
+        ----------
+        path : str
+            Path to the directory containing the Protobuf text file to parse
+        file_name : str
+            Name of the schema file. Defaults to "schema.pbtxt".
+
+        Returns
+        -------
+        TensorflowMetadata
+            Schema object parsed from Protobuf text file
+
+        """
         path = pathlib.Path(path) / file_name
         return cls.from_proto_text(_read_file(str(path)))
 
     def to_proto_text(self) -> str:
+        """Convert this TensorflowMetadata schema object to a Proto text string
+
+        Parameters
+        ----------
+
+        Returns
+        -------
+        str
+            Generated Proto text string
+
+        """
         from tensorflow_metadata.proto.v0 import schema_pb2
 
         return proto_utils.better_proto_to_proto_text(self.proto_schema, schema_pb2.Schema())
 
-    def to_proto_text_file(self, path: str, file_name="schema.pbtxt") -> "TensorflowMetadata":
-        return _write_file(self.to_proto_text(), path, file_name)
+    def to_proto_text_file(self, path: str, file_name="schema.pbtxt"):
+        """Write this TensorflowMetadata schema object to a file as a Proto text string
+
+        Parameters
+        ----------
+        path : str
+            Path to the directory containing the Protobuf text file
+        file_name : str
+            Name of the output file. Defaults to "schema.pbtxt".
+        path: str :
+
+
+        Returns
```

Comment:
Sounds like standards are in flight.  I think Ben suggested omission.  Please let me know either way.  We need it in the CONTRIBUTING doc.